tests/crypto_policy_test.go

```diff
@@ -0,0 +1,240 @@
+package tests
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"net"
+	"net/http"
+	"strconv"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	ocpv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/crypto"
+	core "k8s.io/api/core/v1"
+	apiregv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
+	sspv1beta1 "kubevirt.io/ssp-operator/api/v1beta1"
+	"kubevirt.io/ssp-operator/internal/common"
+	"kubevirt.io/ssp-operator/tests/env"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+var _ = Describe("Crypto Policy", func() {
+	BeforeEach(func() {
+		waitUntilDeployed()
+	})
+
+	AfterEach(func() {
+		strategy.RevertToOriginalSspCr()
+	})
+
+	Context("setting Crypto Policy", func() {
+		table.DescribeTable("Adhere to defined TLSConfig", testSettingTLSConfig,
+			table.Entry("[test_id:???] old", oldPermutation),
+			table.Entry("[test_id:???] intermediate", intermediatePermutation),
+			table.Entry("[test_id:???] modern", modernPermutation),
+			table.Entry("[test_id:???] custom", customPermutation),
+		)
+	})
+})
+
+func operatorPod() core.Pod {
+	cfg, err := config.GetConfig()
+	Expect(err).ToNot(HaveOccurred())
+	apiClient, err = client.New(cfg, client.Options{Scheme: testScheme})
```

Comment:
Can you use the global variable `apiClient` instead of redeclaring it here?

Reply:
sure